app/services/age_calculator.rb

```diff
@@ -0,0 +1,12 @@
+class AgeCalculator
+  def age(date_of_birth, today)
+    # See https://stackoverflow.com/a/2357790
+    if today.month > date_of_birth.month ||
```

Comment:
How about hoisting this guy into a predicate method, something like `had_bithday_this_year?`